repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -235,113 +254,107 @@ def parse_architectures(architectures):
 def is_event_relevant(event, installed_pkgs, tasks):
     """Determine if event is applicable given the installed packages and tasks planned so far."""
     for package in event.in_pkgs.keys():
-        if package in tasks['to_remove']:
+        if package in tasks[Task.remove] and event.action != 'Present':
             return False
-        if package not in installed_pkgs and package not in tasks['to_install']:
+        if package not in installed_pkgs and package not in tasks[Task.install]:
             return False
     return True
 
 
-def add_packages_to_tasks(tasks, packages, key):
-    verb = key[3:].upper()  # 'to_remove' -> 'REMOVE' and so on
+def add_packages_to_tasks(tasks, packages, task_type):
     if packages:
-        api.current_logger().debug('{v} {p}'.format(v=verb, p=', '.join(packages)))
-        tasks[key].update(packages)
+        api.current_logger().debug('{v:7} {p}'.format(v=task_type.name.upper(), p=', '.join(packages)))
```

Comment:
What's the :7 supposed to be doing? By checking the documentation i couldn't figure it out and my test didn't yield anything either, so what's that?

Reply:
Oh, that's just right padding with spaces so that the packages are justified in the log.

Reply:
Oh right, thanks